frontend/src/features/admin-form/create/featureTour/FeatureTour.tsx

```diff
@@ -13,9 +13,15 @@ export const FeatureTour = ({ onClose }: FeatureTourProps): JSX.Element => {
   const [stepIndex, setStepIndex] = useState<number>(0)
   const arrowColor: string = useToken('colors', ['primary.100'])
 
-  const handleJoyrideCallback = (data: CallBackProps) => {
-    const { index, status, type } = data
-
+  const handleJoyrideCallback = ({
+    index,
+    status,
+    type,
+  }: {
+    index: number
+    status: valueof<status>
+    type: string
+  }) => {
```

Comment:
this can still be `CallbackProps` haha, don't need change. you are just destructuring a subset. if you really want to be disciplined about the props you can use the `Pick` type to pick the subset props.

Reply:
Updated as per your comment